README.md

```diff
@@ -36,35 +47,72 @@ The application has the following runtime dependencies:
 * `CUSTOMS_FINANCIALS_HODS_STUB`
 * `CUSTOMS_FINANCIALS_SDES_STUB`
 * `CONTACT_FRONTEND`
- 
-You can use the `CUSTOMS_FINANCIALS_FRONTEND_DEPS` profile in service manager to start up these services without this
-project, to help run locally.
 
-Once these services are running, you should be able to do `sbt "run 9876"` to start in `DEV` mode or 
-`sbt "start -Dhttp.port=9876"` to run in `PROD` mode.
 
-## Running tests
+### Login enrolments
+
+The service can be accessed by using below enrolments and with below sample EORI numbers, via http://localhost:9949/auth-login-stub/gg-sign-in (on local) or https://<host:port>/auth-login-stub/gg-sign-in on DEV/QA/STAGING
+
+Redirect URL - `/customs/payment-records`
+
+| Enrolment Key	| Identifier Name | Identifier Value |
```

Comment:
Please cross check this in Preview mode . Its not displaying as a table. May be it has extra |-----------|. Please check